### DISCUSSION TITLE
Extract logger into separate library

### DISCUSSION THREAD

**Discussion by ilianOptrel** [pschatzmann/TinyFTPClient#11] [Ideas]
Hi
Would it make sense to create a separate library only for the logger? I would like to use it in my own arduino libraries...

**Comment by pschatzmann** [comment]
If I remember right, there are plenty of existing logging Arduino libraries to choose from.

I usually include a simple logging implementation in my projects that build upon (standard C) sprintf() and (Arduino) println() to avoid any dependencies to additional libraries.
